sdks/python/apache_beam/metrics/metricbase.py

```diff
@@ -92,7 +94,14 @@ def dec(self, n=1):
 
 
 class Distribution(Metric):
-  """Distribution Metric interface. Allows statistics about the
-    distribution of a variable to be collected during pipeline execution."""
+  """Distribution Metric interface. Allows statistics about the distribution
+    of a variable to be collected during pipeline execution."""
   def update(self, value):
     raise NotImplementedError
+
+
+class Gauge(Metric):
+  """Gauge Metric interface. Allows to track the latest value of a variable
```

Comment:
<!--new_thread; commit:3b2d3da5e97094e4521c80e07118d80343634698; resolved:0-->
The first line of the docstring should be a one-line comment.  Please add two newlines before "Allows".

grammar: "Allows tracking of the latest ..."